packages/neuron-wallet/src/block-sync-renderer/sync/queue.ts

```diff
@@ -35,13 +35,18 @@ export default class Queue {
 
   private yieldTime = 1
 
-  constructor(url: string, lockHashes: string[], startBlockNumber: bigint) {
+  private multiSignCodeHash: string
+  private multiSignBlake160s: string[]
+
+  constructor(url: string, lockHashes: string[], startBlockNumber: bigint, multiSignCodeHash: string, multiSignBlake160s: string[]) {
```

Comment:
Params order matters (to group things together). Could we move `startBlockNumber` to the end, or before `lockHashes`?

Reply:
Moved to the end